mitogen/master.py

```diff
@@ -189,6 +197,11 @@ def _py_filename(path):
     if os.path.exists(path) and _looks_like_script(path):
         return path
 
+    basepath = os.path.basename(path)
+    for filename in SPECIAL_FILE_PATHS:
+        if basepath == filename:
+            return path
```

Comment:
I think this could be simplified. Is your version more performant somehow?
```suggestion
    if basepath in SPECIAL_FILE_PATHS:
        return path
```

Reply:
:man_facepalming: you're correct! I'll make the SPECIAL_FILE_PATHS a set and compare against basepath as you suggest :sweat_smile: good catch